telemetry/main.schema.json

```diff
@@ -33,6 +33,10 @@
           "type": "string",
           "pattern": "^\\d{2}\\."
         },
+        "displayVersion": {
+          "type": "string",
+          "pattern": "^\\d{2}\\."
```

Comment:
When will Firefox 99 come out? Do we need to worry about 3+ digits? Should we force the `version` field to "X.Y" where X and Y are positive ints?